03-improving-oss-impact/02-effective-upstream-contributions.md

```diff
@@ -85,6 +83,7 @@ Interfacing with downstream projects is an extension of your collaboration strat
 
 - **Answer questions:** Downstream projects are first-line users who will have many good questions and identify several issues in your project. You can work with them to build a knowledge base, improve your project documentation, and gather their feedback to enhance the project.
 - **Communicate changes:** Even minor changes can cause severe challenges downstream, so involve downstream projects in development discussions and proactively share your release notes, release candidates, and more. They can help you thoroughly test your RCs before wide releases.
+<!-- JRG: I would also add something about a solid deprecation policy and Hyrum's law -->
```

Comment:
@jaimergp Would it be alright if I push up a commit to your PR to add this?

Reply:
Of course!